contest/contest_info.py

```diff
@@ -63,10 +63,38 @@ def get_running_contests():
 def get_contestant_list(contest):
     return Contestant.objects.filter(contest = contest)
 
+
+def get_contestant(contest):
+    contestants = Contestant.objects.filter(contest=contest)
+    return [contestant.user for contestant in contestants]
+
+
 def get_total_testcases(problem):
     testcases = Testcase.objects.filter(problem = problem)
     return testcases.count()
 
+
+def get_contest_submissions(contest, submissions):
```

Comment:
How about pass a boolean parameter ```owner_mode``` to this function.
In owner mode, ```filter_end_time``` will always be ```contest.end_time```